product_unavailable/controllers/main.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class SearchOrder(WebsiteSale):
+    def _get_search_order(self, post):
```

Comment:
Is this overriding another method that already exists in `WebsiteSale`? If that's the case, we're breaking inheritance chain. Instead should call the super, catch the result, return a new tuple - although I would return a list instead because it will be easier to generate